source/sessions/tests/README.rst

```diff
@@ -178,7 +178,7 @@ Skip this test if your driver does not allow forking.
 14. Implicit sessions only allocate their server session after a successful connection checkout
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Create a MongoClient with the following options: ``maxPoolSize=1`` and ``retryWrites=true``
+* Create a MongoClient with the following options: ``maxPoolSize=1`` and ``retryWrites=true``. If testing against a sharded deployment, be sure to connect to only a single mongos.
```

Comment:
Should we use the MUST keyword to make it clear that this is a requirement?

For example:
> If testing against a sharded deployment, the test runner MUST ensure that the MongoClient connects to only a single mongos host.

(wording borrowed from the description of `useMultipleMongoses` in the [Unified Test Format](https://github.com/mongodb/specifications/blob/0e2f8f93c9ae2b3ce7227db1d3a370bd9246ebb6/source/unified-test-format/unified-test-format.rst) spec)

Reply:
Yes, I agree with this.